src/clj/rems/util.clj

```diff
@@ -110,3 +110,11 @@
   [dir]
   (.mkdirs dir)
   (delete-directory-contents-recursively dir))
+
+(defn contains-all-kv-pairs? [supermap map]
+  (superset? (set supermap) (set map)))
+
+(defn apply-filters [filters coll]
+  (let [filters (or filters {})]
+    (filter #(contains-all-kv-pairs? % filters) coll)))
```

Comment:
these probably could exist in common namespace as they are usable from cljs too.